Adding updates to the CMake: Quick Start to the existing docs (#7309)
microsoft/vscode-docs: docs/cpp/cmake-linux.md

## Create a CMake project

If you do not have an existing CMake project, follow the steps in [Create a CMake project](cmake-quickstart.md).

> If you already have an existing CMake project that has a `CMakeLists.txt` file in the root directory, skip to [Configure Hello World](#configure-hello-world) to configure your project.

Reviewer: ```suggestion
If you already have an existing CMake project that has a `CMakeLists.txt` file in the root directory, continue to [Configure Hello World](#configure-hello-world) to configure your project.
```

### Configure Hello World

> Before you can use the CMake Tools extension to build a project, you need to configure it to know about the compilers on your system. There are two ways to configure CMake in VS Code: CMake Presets and CMake Kits/Variants.

Reviewer: ```suggestion
Before you can use the CMake Tools extension to build a project, you need to configure it to know about the compilers on your system. There are two ways to configure CMake in VS Code: 

- Use CMake Presets (recommended)
- Use CMake Kits/Variants
```

#### Configure using CMake Presets
We recommend using CMake Presets for managing your CMake configurations. CMake Presets enable you to specify a common JSON file, where you store all the configurations for your project. You can then share this file with others, across different IDEs, and across different operating systems.

If you created a project by following the step in [Create a CMake project](cmake-quickstart), your project is configured to use CMake Presets.

If your project has a `CMakePresets.json` file, you can use the Configure and Build presets for specifying how to build your project on your machine.

You can view the active configuration of presets in the Project Status in the CMake Tools view under the **Configure** and **Build** node. You can select these nodes at any time to set or change your Configure and Build presets.

![CMake Side Panel with presets](images/cpp/cmake-side-panel-presets.png)

You can also set any of your presets by running the **CMake: Select Configure Preset** or **CMake: Select Build Preset** commands in the Command Palette (`kb(workbench.action.showCommands)`).

![CMake Select presets](images/cpp/select-cmake-preset.png)

> #### Configure Using CMake Kits

Reviewer: ```suggestion
#### Configure using CMake Kits
```

If your project does not have a `CMakePresets.json` file, you need to use kits. A kit represents a toolchain, which is the compiler, linker, and other tools used to build your project.

To scan for kits:

1. Open the Command Palette (`kb(workbench.action.showCommands)`) and run **CMake: Select a Kit**. The extension automatically scans for kits on your computer and creates a list of compilers found on your system.

1. Select the compiler you want to use. For example, depending on the compilers you have installed, you might see something like:

    ![Select the kit](images/cpp/cmake-selectkit.png)

> The kit you selected previously is shown in the **Project Status** section in the CMake Tools view. For example:

Reviewer: ```suggestion
    The kit you selected previously is now shown in the **Project Status** section in the CMake Tools view.
```

> ![Selected kit in Sidebar](images/cpp/cmake-kit.png)

Reviewer: ```suggestion
    ![Selected kit in Sidebar](images/cpp/cmake-kit.png)
```

To change the kit, you can select the kit in the **Project Status** section in the CMake Tools view, or run the **CMake: Select a Kit** command again from the Command Palette. If you don't see the compiler you're looking for, you can edit the `cmake-tools-kits.json` file in your project. To edit the file, open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Edit User-Local CMake Kits** command.

> ##### Select a variant

Reviewer: Is this section a subsection of "Configure using CMake Kits"? It's weird to have a single subsection.

Author: I added it to be one large section
